README.md

```diff
@@ -1,11 +1,22 @@
-# Runtime pallet library
+# Runtime Pallet Library
 
-General purpose utilities and pallets for [*Substrate*](https://substrate.io/)
+[![License: LGPL v3](https://img.shields.io/badge/License-LGPL_v3-blue.svg)](https://www.gnu.org/licenses/lgpl-3.0)
+[![Documentation](https://img.shields.io/badge/Rust_Docs-RPL-24CC85?logo=rust)](https://foss3.github.io/runtime-pallet-library/)
 
-## Utilities
+General purpose utilities and pallets for [_Substrate_](https://substrate.io/).
+
+## Components
+
+### Utilities
 
 - [`mock-builder`](mock-builder): Build mock pallets from traits
 
-## Pallets
+### Pallets
+
+None so far.
+
+## Projects using RPL
```

Comment:
```suggestion
## Projects using RPL (Runtime Pallet Library)
```

It took me some seconds to realize what the acronym was (although there can be a lack of coffee today 😆)